ocs_ci/templates/provider-client-deployment/storage_client.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: ocs.openshift.io/v1alpha1
+kind: StorageClient
+metadata:
+  name: storage-client
+  namespace: openshift-storage-client
```

Comment:
StorageClient is cluster scoped from 4.16. So namespace parameter is not required from 4.16. Remove namespace either in template or while creating storage client.

Reply:
Updated in latest commit.